inst/tests/tests.Rraw

```diff
@@ -21593,3 +21593,8 @@ test(2336.3, all.equal(as.Date(t1) - t2, t1 - t2))
 test(2336.4, all.equal(as.Date(t2) - t1, t2 - t1))
 test(2336.5, all.equal(t1 - as.Date(t2), t1 - t2))
 test(2336.6, all.equal(t2 - as.Date(t1), t2 - t1))
+
+# fwrite: allow dec=',' with single column, #7227
+test(2337.1, fwrite(data.table(1), dec = ",", file = nullfile()), NULL)
+test(2337.2, fwrite(data.table(1, 2), dec = ",", sep = ","), error = "(NCOL(x) <= 1L || dec != sep) is not TRUE")
```

Comment:
This depends on the error message produced by `stopifnot()` and is therefore fragile. I recommend skipping this test on R before 4.0.0 where names in `stopifnot()` are ignored.